.drone.star

```diff
@@ -31,7 +31,6 @@ def main(ctx):
       'ldap': 'https://marketplace.owncloud.com/api/v1/apps/user_ldap/0.15.0',
       'ldap_sha': '742703c66a8a112636c30aa500d7b4fdff670bac179fe3ae658a2b4016e5847f',
       'php': '7.3',
-      'behat_version': 'behat-10.4.0',
```

Comment:
This will need to go back - this is the 10.4.0 section, which does need the behat composer.json that is at `behat-10.4.0`

Reply:
The 10.4.0 section has now disappeared, so my comment is not relevant.